clean/ca/redding_pd.py

```diff
@@ -0,0 +1,93 @@
+import re
+import time
+import urllib.parse
+from pathlib import Path
+
+from bs4 import BeautifulSoup
+
+from .. import utils
+from ..cache import Cache
+from .config.chula_vista_pd import index_request_headers
```

Comment:
to avoid confusion, can you rename and update the scrapers using this reusable set of headers?